### PR TITLE
feat: util/log support log to stderr and add log-to-stderr flag for kusion compile

### PR DESCRIPTION
**Pull request by howieyuen**
fix: https://github.com/KusionStack/kusion/issues/75

### CODE REVIEW

**Review by elliotxx — APPROVED** [review]
lgtm

File: pkg/log/log.go [hunk 1]
```diff
@@ -98,3 +98,8 @@ func GetLogger() Logger {
 func With(args ...interface{}) Logger {
 	return log.With(args...)
 }
+
+// LogToStderr sets whether log to stderr instead of files
+func LogToStderr(silent bool) {
```
Comment (elliotxx) [inline pkg/log/log.go:103]: slient is typo?

File: pkg/log/log.go [hunk 2]
```diff
@@ -98,3 +98,8 @@ func GetLogger() Logger {
...
+	log.LogToStderr(silent)
+}
```
Comment (elliotxx) [inline pkg/log/log.go:105]: Consider:
```
func SetOutput(out io.Writer) {
	log.SetOutput(out)
}
```

Also see: https://github.com/sirupsen/logrus/blob/master/logger.go#L391

File: pkg/log/log.go (hunk 1 above)
Comment (howieyuen) [reply pkg/log/log.go]: sorry...

File: pkg/log/log.go (hunk 2 above)
Comment (howieyuen) [reply pkg/log/log.go:105]: Good Idea

**Review by elliotxx — APPROVED** [review]
lgtm

**Review by markliby — APPROVED** [review]

**Pull request merged**